### ISSUE TITLE
screenreader makes the game unable to load

### ISSUE THREAD

**Issue by osboz** [awWhy/Fundamental#10]
the game was working fine until i tried to activate screenreader-mode. 
After i reloaded the page the game became unable to load.

**Comment by awWhy** [comment]
Any errors in console

**Comment by osboz** [comment]
Uncaught TypeError: i.ariaLabel is undefined
    O Stage.ts:1018
    H Update.ts:1058
    Ge Special.ts:666
    xt Main.ts:38
    <anonymous> Main.ts:72
    <anonymous> Main.ts:485
Stage.ts:1018:12
    O Stage.ts:1018
    H Update.ts:1058
    Ge Special.ts:666
    xt Main.ts:38
    <anonymous> Main.ts:72
    <anonymous> Main.ts:485

**Comment by awWhy** [comment]
Fix pushed

**Comment by awWhy** [comment]
It was definetly because Firefox doesn't support Aria shorthands...
